src/style.css

```diff
@@ -2,0 +1,92 @@
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir', Arial, sans-serif;
+}
+
+input {
+  all: unset;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: "Avenir" ;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
```

Comment:
When using the `@font-face` rule, it's important to define a set of font-weights and styles for better text rendering and to avoid faux styles in browsers that do not support the weight or style used in the CSS. Also, consider including a local source if the font is available on the user's system, which can save loading time.